app/src/renderer/vuex/modules/blockchain.js

```diff
@@ -9,10 +12,18 @@ export default ({ node }) => {
     blockMetas: {},
     subscribedRPC: null,
     loading: false,
-    error: null
+    error: null,
+    peers: [],
```

Comment:
what's `peers` for?

Reply:
If you think we won't show them I can remove it, I was thinking would be nice to have that list to show extra informations about the network, even just the versions involved (or a pie-like)

Reply:
interesting. `node_info` could be valuable and is used in a couple places - but how could peer `node_info` be useful?

Reply:
peers give back the node_info of every peer in the network

Reply:
ok actually i think this will be cool. let's leave it in and i'll add something to PageNetwork to make use of this data.